File: lihua-cloud/lihua-biz/lihua-monitor/src/main/java/com/lihua/monitor/model/CpuMonitor.java
package com.lihua.monitor.model;

import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
public class CpuMonitor {
    /**
     * 物理核心数
     */
    private String physicalCores;

    /**
     * 逻辑核心数
     */
    private String logicalCores;

    /**
     * 系统使用率
     */
    private String sysUse;

    /**
     * 用户使用率
     */
    private String userUse;

    /**
     * 等待率
     */
    private String await;

    /**
     * 空闲率
     */
    private String free;
}

File: lihua-vue/src/api/monitor/server/type/cpu-monitor.ts
export interface CpuMonitor {
    /** 物理核心数 */
    physicalCores: string;

    /** 逻辑核心数 */
    logicalCores: string;

    /** 系统使用率 */
    sysUse: string;

    /** 用户使用率 */
    userUse: string;

    /** 等待率 */
    await: string;

    /** 空闲率 */
    free: string;
}